frontend/src/containers/SurveyPage/SurveyPage.tsx

```diff
@@ -213,81 +233,95 @@ const SurveyPage: React.FC<SurveyPageProps> = ({ user_type }) => {
-            {hasSubscriptionIntent && (
-              <Typography variant="body2" className="newsletter-text__small">
-                By subscribing to the newsletter, I have read this form and
-                understand its content and voluntarily give my consent for the
-                collection, use, processing, storage and retention of my
-                personal data or information to Sence1 for the purpose(s)
-                described in the{' '}
-                <Link
-                  style={{ color: 'var(--primary-color)' }}
-                  to={'/privacy-consent'}
-                >
-                  Privacy Policy
-                </Link>{' '}
-                document
-              </Typography>
-            )}
-          </Box>
-        </>
+        <Typography
+          variant="h5"
+          sx={{
+            fontWeight: '700',
+            textAlign: 'center',
+            marginTop: '20px',
+          }}
+        >
+          You are currently not subscribed to our newsletter
+        </Typography>
       )}
       <Survey
         {...{
           surveyQuestions,
           surveyResponses,
           handleChange,
-          handleSubmit,
           subscriber,
           pathname,
-          message,
         }}
       />
+      {!isASubscriber && (
+        <Box>
+          <FormGroup>
+            <FormControlLabel
+              sx={{
+                margin: 'auto',
+                '& .MuiFormControlLabel-label ': {
+                  fontWeight: 700,
+                  fontSize: '24px',
+                  color: 'var(--secondary-color)',
+                },
+              }}
+              control={
+                <Checkbox
+                  sx={{
+                    color: 'var(--secondary-color)',
+                    '&.Mui-checked': {
+                      color: 'var(--secondary-color)',
+                    },
+                  }}
+                  defaultChecked
+                />
+              }
+              label="I want to subscribe"
+              onChange={() => setHasSubscriptionIntent(!hasSubscriptionIntent)}
+            />
+          </FormGroup>
+          {hasSubscriptionIntent && (
+            <Typography variant="body2" className="newsletter-text__small">
+              By subscribing to the newsletter, I have read this form and
+              understand its content and voluntarily give my consent for the
+              collection, use, processing, storage and retention of my personal
+              data or information to Sence1 for the purpose(s) described in the{' '}
+              <Link
+                style={{ color: 'var(--primary-color)' }}
+                to={'/privacy-consent'}
+              >
+                Privacy Policy
+              </Link>{' '}
+              document
+            </Typography>
+          )}
+        </Box>
+      )}
+      <Button
+        variant="contained"
+        color="secondary"
+        onClick={handleSubmit}
```

Comment:
<img width="549" alt="image" src="https://github.com/Sence1-inc/sp-scholaris-master/assets/156179390/f0d3a654-a6e8-439d-b756-1842726c3d4b">
I get the same error message as dashboard deleting